apps/site/next.dynamic.mjs

```diff
@@ -93,7 +93,7 @@ const getDynamicRouter = async () => {
 
     return [...pathnameToFilename.keys()]
       .filter(shouldIgnoreStaticRoute)
-      .concat([...DYNAMIC_ROUTES.keys()]);
+      .concat([...(await DYNAMIC_ROUTES()).keys()]);
```

Comment:
:)

Reply:
change reverted, no longer needed (thanks to work done on the adapter 🙂) 🥳